function-reference/copyToClipboard.md

```diff
@@ -0,0 +1,21 @@
+---
+description: This action is used to copy text to the clipboard
+---
+
+# copyToClipboard
+
+Uses the [copy-to-clipboard](https://www.npmjs.com/package/copy-to-clipboard) library
```

Comment:
This can be mentioned later as part of the argument list so users know what values the options argument takes